dev-packages/node-integration-tests/suites/tracing/linking/test.ts

```diff
@@ -0,0 +1,75 @@
+import { createRunner } from '../../../utils/runner';
+
+// A general note regarding this test:
+// The fact that the trace_id and span_id are correctly linked is tested in a unit test
```

Comment:
I also only learned this by chance but if this is about not being able to store the ids in variables to compare: You can pass a function to expect({ transaction }) so that you can write better assertions than just comparing the matching object. See [here](https://github.com/getsentry/sentry-javascript/blob/5b665b86440a3888b7ce0535d7007f67800769ca/dev-packages/node-integration-tests/suites/tracing/httpIntegration/test.ts#L142-L153) for an example